### PR TITLE
fix(@desktop/wallet): Wallet - Send - Crash after opening send modal

### LINKED ISSUE #6801: Wallet - Send - Crash after opening send modal
# Bug Report

## Description

App crashed after opening wallet and send transaction modal

```qrc:/StatusQ/src/StatusQ/Components/StatusCard.qml:252:9: QML StatusIcon: Cannot open: qrc:/imports/assets/png/network/Network=Tetnet.png
qrc:/StatusQ/src/StatusQ/Components/StatusCard.qml:489:13: QML PropertyChanges: Cannot assign to non-existent property "edit.color"
DBG 2022-08-03 20:17:53.796+01:00 NewBE_callPrivateRPC                       topics="rpc" tid=3039194 file=core.nim:20 rpc_method=wallet_getSuggestedFees
ERR 2022-08-03 20:17:54.065+01:00 rpc response error                         topics="rpc" tid=3039194 file=core.nim:29 err="\nstatus-go error [methodName:wallet_getSuggestedFees, code:-32000, message:transaction type not supported ]\n"
INF 2022-08-03 20:17:54.066+01:00 shutting down...                           topics="status-app" tid=3039194 file=nim_status_client.nim:134
qrc:/imports/shared/popups/SendModal.qml:393:33: QML GasValidator: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qrc:/imports/shared/popups/SendModal.qml:393:33: QML GasValidator: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qrc:/imports/shared/popups/SendModal.qml:393:33: QML GasValidator: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qrc:/imports/shared/popups/SendModal.qml:393:33: QML GasValidator: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
Traceback (most recent call last)
/Users/jenkins/workspace/status-desktop/platforms/macos/src/nim_status_client.nim(168) nim_status_client
/Users/jenkins/workspace/status-desktop/platforms/macos/src/nim_status_client.nim(143) mainProc
/Users/jenkins/workspace/status-desktop/platforms/macos/src/app/boot/app_controller.nim(233) delete
/Users/jenkins/workspace/status-desktop/platforms/macos/src/app/global/global_singleton.nim(67) delete
/Users/jenkins/workspace/status-desktop/platforms/macos/vendor/nimqml/src/nimqml/private/qqmlapplicationengine.nim(40) delete
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
```

## Steps to reproduce

Login
Navigate to wallet
Click on Send transaction

Note: It was not possible to reproduce every time but have seen it happen at least twice now

#### Expected behavior

Open send modal without crashing.

#### Actual behavior

App crashed.

### Additional Information

- Status desktop version: Mac build 198
- Operating System: Mac

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
The error seen in the bug is linked to getSuggestedFees returning an rpc error that was unhandled, I think this should resolve this issue.

fixes #6801

### What does the PR do

The error seen in the bug is linked to getSuggestedFees returning an rpc error that was unhandled, I think this should resolve this issue.

### Affected areas

Wallet

### CODE REVIEW

**Comment by Khushboo-dev-cpp** [comment]
> What is the error?

```
qrc:/StatusQ/src/StatusQ/Components/StatusCard.qml:489:13: QML PropertyChanges: Cannot assign to non-existent property "edit.color"
DBG 2022-08-03 20:17:53.796+01:00 NewBE_callPrivateRPC                       topics="rpc" tid=3039194 file=core.nim:20 rpc_method=wallet_getSuggestedFees
ERR 2022-08-03 20:17:54.065+01:00 rpc response error                         topics="rpc" tid=3039194 file=core.nim:29 err="\nstatus-go error [methodName:wallet_getSuggestedFees, code:-32000, message:transaction type not supported ]\n"
INF 2022-08-03 20:17:54.066+01:00 shutting down...                           topics="status-app" tid=3039194 file=nim_status_client.nim:134
qrc:/imports/shared/popups/SendModal.qml:393:33: QML GasValidator: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qrc:/imports/shared/popups/SendModal.qml:393:33: QML GasValidator: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qrc:/imports/shared/popups/SendModal.qml:393:33: QML GasValidator: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qrc:/imports/shared/popups/SendModal.qml:393:33: QML GasValidator: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
Traceback (most recent call last)
/Users/jenkins/workspace/status-desktop/platforms/macos/src/nim_status_client.nim(168) nim_status_client
/Users/jenkins/workspace/status-desktop/platforms/macos/src/nim_status_client.nim(143) mainProc
/Users/jenkins/workspace/status-desktop/platforms/macos/src/app/boot/app_controller.nim(233) delete
/Users/jenkins/workspace/status-desktop/platforms/macos/src/app/global/global_singleton.nim(67) delete
/Users/jenkins/workspace/status-desktop/platforms/macos/vendor/nimqml/src/nimqml/private/qqmlapplicationengine.nim(40) delete
SIGSEGV: Illegal storage access. (Attempt to read from nil?)

this is what was reported here https://github.com/status-im/status-desktop/issues/6801
```

File: src/app_service/service/transaction/service.nim [hunk 1]
```diff
@@ -370,22 +370,28 @@ QtObject:
...
-    return SuggestedRoutes(
-      networks: Json.decode($response.result{"networks"}, seq[NetworkDto])
-    )
+    try:
+      let response = eth.suggestedRoutes(account, amount, token, disabledChainIDs)
+      return SuggestedRoutes(
+        networks: Json.decode($response.result{"networks"}, seq[NetworkDto])
+      )
+    except Exception as e:
+      error "Error getting suggested fees", msg = e.msg
```
Comment (borismelnik) [inline src/app_service/service/transaction/service.nim:394]: copy-paste typo :) `Error getting suggested routes :)

**Review by borismelnik — CHANGES_REQUESTED** [review]
LGTM, just a small typo :)

File: src/app_service/service/transaction/service.nim (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply src/app_service/service/transaction/service.nim]: done @borismelnik !

**Review by borismelnik — APPROVED** [review]
LGTM

**Pull request merged**